presto-main/src/main/java/com/facebook/presto/sql/planner/sanity/TypeValidator.java

```diff
@@ -184,10 +184,15 @@ private void checkFunctionSignature(Map<Symbol, Aggregation> aggregations)
             }
         }
 
-        private void checkFunctionCall(Map<Symbol, Aggregation> aggregations)
+        private void checkAggregation(Map<Symbol, Aggregation> aggregations)
```

Comment:
`testInvalidAggregationFunctionCall` will fail

Reply:
This test will be disabled temporarily until we converted to CallExpression.